Mage.Sets/src/mage/cards/s/SphereOfSafety.java

```diff
@@ -51,7 +52,7 @@ public SphereOfSafety copy() {
 class SphereOfSafetyPayManaToAttackAllEffect extends CantAttackYouUnlessPayAllEffect {
 
     SphereOfSafetyPayManaToAttackAllEffect() {
-        super(null, true);
+        super(Duration.WhileOnBattlefield, new ManaCostsImpl<>("{X}"), Scope.YOU_AND_CONTROLLED_PLANESWALKERS);
```

Comment:
So the cost in the constructor is just for the text generation right? The override still works as it should?

Reply:
Yes, half the effects counting something were using `null` and the other half this fake `ManaCostsImpl<>("{X}")`` that is not used in reality.

I forgot to remove the null check for Cost in the default staticText of the effect.